reference/library/src/morpheus-master/sass/modules/tool/_base.scss

```diff
@@ -59,8 +59,7 @@
 		float: right;
 		display: block;
 		position: relative;
-		margin: 0px 0 -20px 0;
-		top: 9px;
+		margin: -1em 0 0 0;
```

Comment:
Yes, please, could you do that update and then squash your commits again? Thanks :)